preparing-kubernetes-cluster.html.md.erb

```diff
@@ -52,12 +52,33 @@ for <%= vars.app_runtime_short %> on TKG:
 * Your Tanzu Kubernetes Grid environment:
     * Is Tanzu Kubernetes Grid version is 1.0 or later.
     * Includes `metrics-server`. TKG does not bundle `metrics-server`,
-    which is required by Tanzu Application Service.
-    To deploy `metrics-server`, run:
+      which is required by Tanzu Application Service.
+      To deploy `metrics-server`, run:
 
         ```
-        kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.3.6/components.yaml
+        kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.4.1/components.yaml
         ```
+
+      * The above apply will likely fail on Tanzu Kubernetes Grid. To resolve the issue, you'll need to add the
```

Comment:
I wonder if there are any specific versions we could mention for this?

Reply:
@tcdowney It isn't clear to me if there are any current versions of TKG where this won't fail. I haven't heard back from the TKG team yet about their fix for this, but they would know. Perhaps we could at least say it was observed in the specific version of TKGs that we're using.